composer.json

```diff
@@ -15,17 +15,17 @@
         }
     ],
     "require": {
-        "php": ">=5.5.0",
+        "php": ">=7.0",
         "fzaninotto/faker": "~1.0",
-        "laravel/framework": "~5.4",
```

Comment:
The package registers artisan commands, blade views and other assets. I don't think switching the dependency from the framework to http only covers all these.

Reply:
@shalvah Dependencies are not defined explicitly I know, because you cannot do this: "laravel || lumen". But we know that this package is installed alongside lumen/laravel so we know that the necessary packages are installed.
Another solution is try to define the explicit illuminate packages, but I think the current solution is enough.

Reply:
I think it's better to define the explicit packages. When it comes to dependencies, explicit is better than implicit. Plus, a contributor who clones this repo won't be able to develop (and CI will fail as well) because Composer won't know to automagically install the rest of the framework.

Reply:
@gazben I just realised I can't merge this to master in its current state. I'm not fully convinced about the dependencies yet (admittedly, I haven't had time to review it in depth), but more importantly, it's a breaking change (PHP version support). I've been drafting out a list of features for version 3 of this library, with Lumen support there and support for only L5.5+. I'll post it in a new issue and tag you.
We'd also need to maintain a separate 2.x branch for compatibility issues.
I appreciate your contribution and willingness to resolve this👍

Reply:
@shalvah The php version change is breaking but I don't think anyone is using php5.6 anymore. Even the dependencies require php7.0 now. I spent a good two hour resolving the issues and I honestly can't find the answer why are the test failing, so sorry about that. On a sidenote: the package is not testable on windows because of the rm command in the tests, that should use php commands. 

Tag me please. I will continue using my fork.